inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_roi_feature_extractor_test.hpp

```diff
@@ -89,9 +89,6 @@ TEST_P(myriadLayersTestsROIFeatureExtractor_smoke, ROIFeatureExtractor) {
         {"output_size",     std::to_string(test_params.output_size)},
         {"sampling_ratio",  std::to_string(test_params.sampling_ratio)},
         {"pyramid_scales",  pyramid_scales_str},
-        {"distribute_rois_between_levels", "1"},
-        {"preserve_rois_order", "1"},
-        {"image_id", "0"},
```

Comment:
@gladilov-gleb , please, double check that these attributes are not supported in the MYX/KMB plugins.

Reply:
MKLDNN implementation doesn't use these attributes. See lines 324-327 of https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/roifeatureextractor_onnx.cpp.

Reply:
According to lines 107-109 of https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/vpu/graph_transformer/src/stages/roi_feature_extractor.cpp, these attributes are not used in VPU.